src/promptflow-evals/promptflow/evals/synthetic/simulator/_conversation/conversation_bot.py

```diff
@@ -0,0 +1,151 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+import logging
+from typing import Dict, List, Tuple, Union
+
+import jinja2
+
+from promptflow.evals.synthetic.simulator._model_tools import (
+    LLAMAChatCompletionsModel,
+    LLMBase,
+    OpenAIChatCompletionsModel,
+    RetryClient,
+)
+
+from .constants import ConversationRole
+from .conversation_turn import ConversationTurn
+
+
+class ConversationBot:
+    def __init__(
+        self,
+        role: ConversationRole,
+        model: Union[LLMBase, OpenAIChatCompletionsModel],
+        conversation_template: str,
+        instantiation_parameters: Dict[str, str],
+    ):
+        """
+        Create a ConversationBot with specific name, persona and a sentence that can be used as a conversation starter.
+
+        Parameters

```

Comment:
why use a different style of paramter doc here?

Reply:
fixed it. This was copied over from the previous repo (like every other file in the simulator). `ConversationBot` is not a public facing class and is only used internally.